src/NetAnalyzers/CSharp/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,11 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+
+### New Rules
```

Comment:
Lets avoid adding new rules to `CSharp/AnalyzerReleases.Unshipped.md`. Can you instead create a base type with the diagnostic descriptors in the [core](https://github.com/dotnet/roslyn-analyzers/tree/main/src/NetAnalyzers/Core) language-agnostic NetAnalyzers project? This way all the shipped/unshipped rules get added to the same files: `src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md` and `src/NetAnalyzers/Core/AnalyzerReleases.Shipped.md`